web/experimental/styles_dark.scss

```diff
@@ -0,0 +1,58 @@
+/* styles_dark.css */
+@import 'package:dart_pad/experimental/scss/colors';
+
+body {
+  background-color: $dark-code-background-color;
+}
+
+.busy {
+  background: url(moving_lines_dark.gif) repeat;
+  box-shadow: inset 0 0 0 1000px rgba(27, 29, 33, .8); /* #1b1e22 */
```

Comment:
`styles_light` uses a variable for the shadow color. Any reason it's hardcoded here for the dark theme rather than something like `$dark-busy-background-color`?